projects/1.18/assets/astral-dimension/astral_dimension/lang/zh_cn.json

```
@@ -0,0 +1,536 @@
+  "item.astral_dimension.fiery_cow_spawn_egg": "烈焰牛刷怪蛋",
+  "item.astral_dimension.fiery_hummingbird_spawn_egg": "烈焰蜂鸟刷怪蛋",
+  "item.astral_dimension.firestorm_spawn_egg": "烈焰风暴刷怪蛋",
+  "item.astral_dimension.fried_zombie_spawn_egg": "熔岩僵尸刷怪蛋",
+  "item.astral_dimension.ghost_jelly": "幽灵凝胶",
+  "item.astral_dimension.ghost_jellyfish_spawn_egg": "幽魂水母刷怪蛋",
+  "item.astral_dimension.golden_bone": "黄金之骨",
+  "item.astral_dimension.golden_skull_spawn_egg": "黄金颅骨刷怪蛋",
+  "item.astral_dimension.harsh_backpack": "粗粝背包",
+  "item.astral_dimension.harsh_leather": "粗粝皮革",
+  "item.astral_dimension.harsh_leather_armor_boots": "粗粝皮革靴子",
+  "item.astral_dimension.harsh_leather_armor_chestplate": "粗粝皮革外套",
+  "item.astral_dimension.harsh_leather_armor_helmet": "粗粝皮革帽子",
+  "item.astral_dimension.harsh_leather_armor_leggings": "粗粝皮革裤子",
+  "item.astral_dimension.keeper_spawn_egg": "守卫刷怪蛋",
+  "item.astral_dimension.locked_visage_spawn_egg": "带枷巨面刷怪蛋",
+  "item.astral_dimension.lunar_beetroot": "月球甜菜",
+  "item.astral_dimension.melee_astral_golem_spawn_egg": "星体傀儡刷怪蛋",
+  "item.astral_dimension.moonbubble": "月亮泡泡",
+  "item.astral_dimension.nether_visage_portal_spawn_egg": "下界传送之面刷怪蛋",
+  "item.astral_dimension.obsidian_bucket": "黑曜石桶",
+  "item.astral_dimension.outer_debris": "界外残骸",
+  "item.astral_dimension.overworld_visage_portal_spawn_egg": "主世界传送之面刷怪蛋",
+  "item.astral_dimension.pigail_no_shell_spawn_egg": "蜗猪刷怪蛋",
+  "item.astral_dimension.pigail_spawn_egg": "蜗猪刷怪蛋",
+  "item.astral_dimension.poisonous_vines_spawn_egg": "毒藤刷怪蛋",
+  "item.astral_dimension.postrum_eye_mob_spawn_egg": "怪奇之眼刷怪蛋",
+  "item.astral_dimension.ranged_astral_golem_spawn_egg": "带电星体傀儡刷怪蛋",
+  "item.astral_dimension.raw_astranite": "粗星钢",
+  "item.astral_dimension.red_chilli": "红辣椒",
+  "item.astral_dimension.scourge_scales": "灾祸之鳞",
+  "item.astral_dimension.sentry_projectile": "哨兵射弹",
+  "item.astral_dimension.sludge_brick": "污泥砖",
+  "item.astral_dimension.slurker_spawn_egg": "潜匿者刷怪蛋",
+  "item.astral_dimension.soul_scythe": "灵魂镰刀",
+  "item.astral_dimension.sticky_porkchop": "黏糊糊的猪排",
+  "item.astral_dimension.storm_wisp_spawn_egg": "风暴游魂刷怪蛋",
+  "item.astral_dimension.super_void_charge": "Super Void Charge",
+  "item.astral_dimension.thunder_clock": "雷霆钟",
+  "item.astral_dimension.thunder_shell": "雷霆螺壳",
+  "item.astral_dimension.titan_sentry_spawn_egg": "泰坦哨兵刷怪蛋",
+  "item.astral_dimension.tormented_boots": "痛苦靴子",
+  "item.astral_dimension.tormented_chestplate": "痛苦胸甲",
+  "item.astral_dimension.tormented_helmet": "痛苦骨盔",
+  "item.astral_dimension.tormented_leggings": "痛苦护腿",
+  "item.astral_dimension.totem_of_decay": "衰败图腾",
+  "item.astral_dimension.void_boots_boots": "虚空靴子",
+  "item.astral_dimension.void_boots_chestplate": "虚空靴子",
+  "item.astral_dimension.void_boots_helmet": "泰坦头盔",
+  "item.astral_dimension.void_boots_leggings": "虚空靴子",
+  "item.astral_dimension.void_centipede_spawn_egg": "虚空蜈蚣刷怪蛋",
+  "item.astral_dimension.void_charge_titan": "Void Charge",
```

大概是某种子弹？

---

```suggestion
  "item.astral_dimension.void_charge_titan": "虚空弹",
```
见原版`火焰弹`/`Fire Charge`